src/account/ModuleManagerInternals.sol

```diff
@@ -244,9 +244,9 @@ abstract contract ModuleManagerInternals is IModularAccount {
 
                 continue;
             }
-            // Hook is a permission hook
-            if (!_validationData.permissionHooks.add(toSetValue(hookConfig))) {
-                revert PermissionAlreadySet(moduleEntity, hookConfig);
+            // Hook is an execution hook
+            if (!_validationData.executionHooks.add(toSetValue(hookConfig))) {
+                revert ExecutionHookAlreadySet(moduleEntity, hookConfig);
```

Comment:
Do we need a custom error for this?
The selector associated ones don't have one. Should we either add for them or use the default one here too?

Reply:
good catch! switched to use the internal function instead. I think we can remove `ValidationFunction` from the error since for `installValidation` its obvious which val function it is, and for installing a module its possible to find that info from the manifest. I think this should cut down code size